File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/membrii-afiliati-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";
import { DocumentIdentifierModel } from "./document-identifier.model ";

@JsonObject
export class MembriiAfiliatiReportFilterModel {

	@JsonProperty("institutieId", Number)
    public institutieId: number = null;

    @JsonProperty("comisieId", Number)
    public comisieId: number = null;

	@JsonProperty("dataSedintaDeLa", JsonDateConverter)
	public dataSedintaDeLa: Date = null;

	@JsonProperty("dataSedintaPanaLa", JsonDateConverter)
    public dataSedintaPanaLa: Date = null;
    
    @JsonProperty("numeReprezentantAcreditat", String)
    public numeReprezentantAcreditat: string = null;
    
    @JsonProperty("prenumeReprezentantAcreditat", String)
    public prenumeReprezentantAcreditat: string = null;
       
	@JsonProperty("numeReprezentantInlocuitor", String)
	public numeReprezentantInlocuitor: string = null;
    
	@JsonProperty("prenumeReprezentantInlocuitor", String)
	public prenumeReprezentantInlocuitor: string = null;

    @JsonProperty("document", DocumentIdentifierModel)
    public document: DocumentIdentifierModel = null;
    
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/MembriiAfiliatiReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class MembriiAfiliatiReportFilterModel {

	private DocumentIdentifierModel document;
	private Long institutieId;
	private String numeReprezentantAcreditat;
	private String prenumeReprezentantAcreditat;
	private String numeReprezentantInlocuitor;
	private String prenumeReprezentantInlocuitor;
	private Date dataSedintaDeLa;
	private Date dataSedintaPanaLa;
	private Integer comisieId;

	public DocumentIdentifierModel getDocument() {
		return document;
	}

	public void setDocument(DocumentIdentifierModel document) {
		this.document = document;
	}

	public Long getInstitutieId() {
		return institutieId;
	}

	public void setInstitutieId(Long institutieId) {
		this.institutieId = institutieId;
	}

	public String getNumeReprezentantAcreditat() {
		return numeReprezentantAcreditat;
	}

	public void setNumeReprezentantAcreditat(String numeReprezentantAcreditat) {
		this.numeReprezentantAcreditat = numeReprezentantAcreditat;
	}

	public String getPrenumeReprezentantAcreditat() {
		return prenumeReprezentantAcreditat;
	}

	public void setPrenumeReprezentantAcreditat(String prenumeReprezentantAcreditat) {
		this.prenumeReprezentantAcreditat = prenumeReprezentantAcreditat;
	}

	public String getNumeReprezentantInlocuitor() {
		return numeReprezentantInlocuitor;
	}

	public void setNumeReprezentantInlocuitor(String numeReprezentantInlocuitor) {
		this.numeReprezentantInlocuitor = numeReprezentantInlocuitor;
	}

	public String getPrenumeReprezentantInlocuitor() {
		return prenumeReprezentantInlocuitor;
	}

	public void setPrenumeReprezentantInlocuitor(String prenumeReprezentantInlocuitor) {
		this.prenumeReprezentantInlocuitor = prenumeReprezentantInlocuitor;
	}

	public Date getDataSedintaDeLa() {
		return dataSedintaDeLa;
	}

	public void setDataSedintaDeLa(Date dataSedintaDeLa) {
		this.dataSedintaDeLa = dataSedintaDeLa;
	}

	public Date getDataSedintaPanaLa() {
		return dataSedintaPanaLa;
	}

	public void setDataSedintaPanaLa(Date dataSedintaPanaLa) {
		this.dataSedintaPanaLa = dataSedintaPanaLa;
	}

	public Integer getComisieId() {
		return comisieId;
	}

	public void setComisieId(Integer comisieId) {
		this.comisieId = comisieId;
	}

}
